components/statusLabel/theme.css

```diff
@@ -0,0 +1,57 @@
+@import '@teamleader/ui-colors';
+@import '@teamleader/ui-typography';
+@import '@teamleader/ui-utilities';
+
+.label {
+  background: var(--background-color);
+  color: var(--text-color);
+  display: inline-block;
+  font-family: var(--font-family-semi-bold);
+  font-weight: 500;
+  margin: auto var(--spacer-smallest);
+  vertical-align: middle;
+}
+
+.mint {
+  --background-color: var(--color-mint-lightest);
+  --text-color: var(--color-mint-darkest);
+}
+
+.violet {
+  --background-color: var(--color-violet-light);
+  --text-color: var(--color-violet-darkest);
+}
+
+.teal {
+  --background-color: var(--color-teal-light);
+  --text-color: var(--color-teal-darkest);
+}
+
+.ruby {
+  --background-color: var(--color-ruby-light);
+  --text-color: var(--color-ruby-darkest);
+}
+
+.gold {
+  --background-color: var(--color-gold-light);
+  --text-color: var(--color-gold-darkest);
+}
+
+.aqua {
+  --background-color: var(--color-aqua-light);
+  --text-color: var(--color-aqua-darkest);
+}
+
+.medium {
+  font-size: calc(1.4 * var(--unit));
+  line-height: calc(1.8 * var(--unit));
+  padding: var(--spacer-smallest) var(--spacer-smaller);
+}
+
+.small {
+  font-size: calc(1.2 * var(--unit));
+  font-variant: small-caps;
+  line-height: calc(1.8 * var(--unit));
+  padding: 0 var(--spacer-smaller);
+  text-transform: uppercase;
```

Comment:
I actually don't see the `text-transform` and `font-variant` in the design files.

Reply:
I talked to Stephen, and you already discussed this apparently and decided to go with the uppercase and small-caps. So ignore my comment 👍

Reply:
The design in Zeplin has been updated by @stephen-tl. But @lowiebenoot you're right about the text-transform which isn't needed in this case. We will just use the **font-variant: small-caps;** to get the right result.